Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail.tutorial

```diff
@@ -2,14 +2,14 @@
   @Intro(title: "Testing the sync-up detail") {
     The `SyncUpDetail` feature has slowly become quite complex. It now handles two forms of 
     navigation (an alert and sheet), it models navigation state with a single enum, and it
-    communicates with a parent feature via delegate actions. It's about time we got some test
+    updates the parent feature via a shared property. It's about time we got some test
     coverage on this feature so we can be sure it works as we expect, and so that we can make future
     changes with confidence.
   }
   
   @Section(title: "Testing the edit flow") {
     @ContentAndMedia {
-      Let's start by testing the edit flow. We will exercise the full user flow of tapping the
+      Let's write a test for the edit flow. We will exercise the full user flow of tapping the
```

Comment:
📓 I've slightly changed this intro which were implying this is the first section of several which is not the case anymore.